src/abstractdataframe/abstractdataframe.jl

```diff
@@ -229,6 +229,13 @@ Base.setproperty!(df::AbstractDataFrame, col_ind::Symbol, x) = setindex!(df, x,
 # Private fields are never exposed since they can conflict with column names
 Base.propertynames(df::AbstractDataFrame, private::Bool=false) = names(df)
 
+"""
+    column(df::AbstractDataFrame, col_ind::Union{Symbol, Signed, Unsigned})
```

Comment:
`Union{Symbol, Signed, Unsigned}` isn't used anywhere currently, and it's not user-friendly. Maybe we should cheat a bit and say `Integer`?

Can you remind me why we don't allow `Bool` here? It works for single indices with arrays, e.g. `(1:3)[true] == 1`.

Reply:
OK. `(1:3)[true]` is a bug that probably should be reported. Here is the expected behavior AFAIK:
```
julia> x = [1,2,3]
3-element Array{Int64,1}:
 1
 2
 3

julia> x[true]
ERROR: ArgumentError: invalid index: true of type Bool
```

Reply:
Are you sure? There's a problem when indexing with a vector containing `Bool` values since there's an ambiguity with logical indexing, but in the scalar case there's no problem and it makes the implementation simpler (it can just use `Integer`).

Reply:
I have opened an issue in Julia for this. I am pretty sure. I index `x` above using a scalar `Bool`.

Reply:
OK I hadn't realized that what you showed was the current behavior. Then indeed the inconsistency is probably a mistake.